<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
 Released as open source by Gabriel Caudrelier

 Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com

 https://github.com/metrodango/pip3line

 Released under BSD 3-Clause see LICENSE for more information
 </comment>
 <class>DistormWidget</class>
 <widget class="QWidget" name="DistormWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="maxInstructionLabel">
     <property name="text">
      <string>Max decoded intruction</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="maxInstructionSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="asmTypeLabel">
     <property name="text">
      <string>ASM type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="asmTypeComboBox">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="offsetLabel">
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="offsetLineEdit">
     <property name="toolTip">
      <string>Offset in hexadecimal</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="showOffsetCheckBox">
     <property name="text">
      <string>Show offset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="showOpcodesCheckBox">
     <property name="text">
      <string>Show opcodes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
